src/UserStory/Entries.js

```diff
@@ -1,23 +1,73 @@
 import React, { Component } from 'react'
 import { Link } from 'react-router-dom'
+import validate from 'validate.js'
+
+const initialState = {
+    date: "",
+    owner: "",
+    usId: "",
+    title: "",
+    question: "",
+    note: "",
```

Comment:
should enclosure the user-story to its own object. Ex:
```
{
    userStory: { date, owner, ... }
}
```

Reply:
When we enclose as such, does it still hold an "" or null value?

And all the variables inside became undefined after doing so. How do i fix this?